src/com_tjvendors/site/views/vendor/view.html.php

```diff
@@ -111,9 +113,73 @@ public function display($tpl = null)
 			$this->options[] = HTMLHelper::_('select.option', $id, $value);
 		}
 
+		// Show or hide vendor information field(s). do this is layout file
+		$showSelectedFields = $this->params->get('show_selected_fields', 0);
+
+		if ($showSelectedFields == 1)
+		{
+			$billingInfoFields = $this->params->get('vendor_field');
+
+			if (isset($billingInfoFields))
+			{
+				foreach ($billingInfoFields as $field)
+				{
+					switch ($field)
```

Comment:
instead of switch can we do like

```php
<?php 
foreach ($billingInfoFields as $field)
{
    $this->vendor_show_ . $field = 1;
}
```

and use as similar loop in layout too instead of hardcoded if checks?